client/src/extension.ts

```diff
@@ -7,13 +7,13 @@ export function activate(context: ExtensionContext) {
   // The server is implemented in node
   let serverModule = context.asAbsolutePath(path.join('server', 'server.js'));
   // The debug options for the server
-  let debugOptions = { execArgv: ["--nolazy", "--debug=6004"] };
+  let debugOptions = { execArgv: ["--nolazy", "--debug=6009"] };
 
   // If the extension is launched in debug mode then the debug server options are used
   // Otherwise the run options are used
   let serverOptions: ServerOptions = {
     run : { module: serverModule, transport: TransportKind.ipc },
-    debug: { module: serverModule, transport: TransportKind.ipc /* *, options: debugOptions /* */ }
```

Comment:
@chuckjaz As I understood it, the debug options do not affect the extension when it's running without the debug flag. I'm happy to add the comment-toggle back in, but I was curious to the reasoning behind this decision.